skranger/ensemble/ranger_forest_classifier.py

```diff
@@ -295,3 +297,28 @@ def predict_log_proba(self, X):
         """
         proba = self.predict_proba(X)
         return np.log(proba)
+
+    def get_importance_pvalues(self):
+        """Return p-values for variable importance using the fast method from Janitza et al. (2016)
+        """
+
+        if self.importance != "impurity_corrected":
+            raise Exception("P-values can only be calculated for impurity corrected importance values")
+
+        vimp = np.array(self.ranger_forest_["variable_importance"])
+        m1 = vimp[vimp < 0]
+        m2 = vimp[vimp == 0]
+
+        if len(m1) < 1:
+            raise Exception("No negative importance values found, cannot calculate p-values.")
+        if len(m2) < 1:
+            vimp_dist = np.concatenate((m1, -m2))
```

Comment:
in this case is vimp_dist just equal to m1?

Reply:
That should be -m1 not -m2, since you're right it would just be m1 in that case.